test/fixture/std-esm/.esmrc

```diff
@@ -0,0 +1,4 @@
+{
+	"cjs": true,
+	"esm": "js"
+}
```

Comment:
👆 You can shorthand to just `"cjs"` in the .esmrc file. In the next release you can use `{ "esm":"cjs" }` as a shorthand for the above too.

Reply:
Like `"cjs"\n` is the full contents of the file?

Reply:
Ya, kinda gross but it works. You may want to wait for the other shorthand form.